app/src/pages/prompt/PromptChatMessagesCard.tsx

```diff
@@ -87,15 +105,21 @@ function ChatMessages({
   const { messages } = template;
   return (
     <Flex direction="column" gap="size-200">
-      {messages.map((message, i) => (
-        // TODO: Handle JSON content for things like tool calls
-        <ChatTemplateMessage
-          key={i}
-          role={message.role as string}
-          content={message.content || JSON.stringify(message.jsonContent)}
-          templateFormat={templateFormat}
-        />
-      ))}
+      {messages.map((message, i) => {
+        const textPart = message.content.find(isTextPart);
+        // TODO(apowell): Break out into switch statement, rendering each message part type
+        if (!isTextPart(textPart)) {
+          return null;
+        }
```

Comment:
Usually I don't care about DRY code but it does feel a bit repetitive ;)

Reply:
Yes all of these text part comments will be fixed in an upcoming PR, once this merges in